recipes/flash/meta.yaml

```diff
@@ -21,13 +24,10 @@ requirements:
 
 test:
   commands:
-    - flash -v
+    - flash2 -v
 
 about:
-  home: https://ccb.jhu.edu/software/FLASH/
+  home: https://github.com/dstreett/FLASH2
   license: GPLv3+
-  summary: 'Merge mates from fragments that are shorter than twice the read length'
+  summary: 'Merge paired-end reads from fragments that are shorter than twice the read length'
 
-extra:
```

Comment:
Why have you removed this? its still the same tool isn't it?
Is there a publication which we can reference?